includes/class-wcmp-frontend-settings.php

```diff
@@ -1,156 +1,150 @@
 <?php
 
-use WPO\WC\MyParcel\Compatibility\WC_Core as WCX;
-use WPO\WC\MyParcel\Compatibility\Order as WCX_Order;
-use WPO\WC\MyParcel\Compatibility\Product as WCX_Product;
```

Comment:
Zijn deze 3 niet meer nodig?

Reply:
Werden überhaupt niet gebruikt in dit bestand